server.js

```diff
@@ -3951,6 +3951,42 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Snap CI build integration.
+// https://snap-ci.com/snap-ci/snap-deploy/branch/master/build_image
+camp.route(/^\/snap(-ci?)\/([^\/]+\/[^\/]+)(?:\/(.+))\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var userRepo = match[2];
+  var branch = match[3];
+  var format = match[4];
+  var url = 'https://snap-ci.com/' + userRepo + '/branch/' + branch + '/build_image.svg';
```

Comment:
No it isn't. Snap CI doesn't have badge of "default" branch because Snap CI is branch based CI. For example, Snap CI doesn't track new branch in the standard, and settings is splitted.